js/chat_app/src/components/NewUserForm/NewUserForm.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Stack,
+  TextField,
+} from "@mui/material";
+import { UsersList } from "../UsersList/UsersList";
+
+const adminId = "c8e26274-93ee-4acb-9f51-126264adaeb2";
+
+export const NewUserForm = ({ open, onClose }) => {
+  const [userName, setUserName] = useState("");
+  const [userPassword, setUserPassword] = useState("");
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      fetch("/api/users", {
+        headers: {
+          "Content-Type": "application/json",
+          "auth-token": adminId,
+        },
+      })
+        .then((response) => response.json())
+        .then((data) => setUsers(data));
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, []);
```

Comment:
I've already seen this function in another component (newChatForm), the difference is only in "auth-token", as far as I understood. Maybe we can make a general function helper for requests and pass in fetch two parameters - our URL and also 'options'. 
For example, `
const request = return fetch(URL, options).then(response => response.json())

const options = {
  method: 'POST', // 'GET', 'POST', 'PATCH', 'PUT', 'DELETE' if needed
  headers: {
    'Content-Type': 'application/json,
    'auth-token': adminId or currentUserToken,
  },
};
`